.github/workflows/ci.yml

```diff
@@ -57,6 +57,7 @@ jobs:
         with: {node-version: "${{ env.NODE_VERSION }}"}
       - run: npm install
       - run: npm run lint-spec
+      - run: npm run lint-hrx
```

Comment:
Why create a separate entrypoint for this rather than just defining it as effectively an additional kind of lint in `lint-spec`?

Reply:
it was mostly to avoid touching the reporter which had some quirks when used on 2 linters, but I can merge it